ranges/intersects.js

```diff
@@ -1,5 +1,7 @@
 const Range = require('../classes/range')
+const parseOptions = require('../internal/parse-options')
```

Comment:
Do we really need to add this to the ranges modules?  With the exception of subset, none of the code there evaluates options, it simply passes it through.

I suppose if parseOptions is optimized enough that re-runs of it are cheaper than initial ones, we could see a net benefit by doing it up front like this.  It's at least worth considering what benefit this gives.

Reply:
With the assumption of faster `parseOptions` indeed this call might be redundant, should I remove all those calls that `parseOptions` and just forward to Range/Semver?

Reply:
Yeah I think for now let's only call it if we actually interact with options directly.  We can revisit this decision in the future if `parseOptions` ever becomes a validator.